instructor/common.py

```diff
@@ -261,10 +261,6 @@ def assignments_variables(request):
     if request.method == "POST":
         form = AssignmentForm(request.POST, instance=assignment)
         if form.is_valid():
-            # If form has changed want to delete existing Treatments
```

Comment:
I think it would be fine to delete and recreate the objects, the only data being lost is which strain treatment combinations the user selected.